find-index.js

```diff
@@ -0,0 +1,11 @@
+export const findIndex = (arr, callback) => {
+  let newArr = [];
+
+  for (let index = 0; index < arr.length; index++) {
```

Comment:
taking a further look at this function, it seems like it's not fully working in the way that findIndex should.   

It should return the first index of what is truthy from your callback, otherwise returns -1 if not found.   So in your test, you're returning every index that is true, instead of the first.   

I'd recommend playing with the demo [here on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) and looking at how this function behaves to solve this.